src/components/StickyHeader/StickyHeader.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import styles from './StickyHeader.scss';
+
+interface Props extends React.HTMLAttributes<HTMLDivElement> {
+    /** Node(s) to be rendered inside the container */
+    children: ReactNode;
+    /** Node to be rendered as a sticky header */
+    header: SingleReactNode;
+    /** CSS class name that will be applied to the header element */
+    headerClassName?: string;
+}
+
+const { block, elem } = bem('StickyHeader', styles);
+
+export const StickyHeader: React.FC<Props> = (props) => {
+    const { children, header, headerClassName = '', ...rest } = props;
+    return (
+        <div {...block()} {...rest}>
+            <div
+                {...elem('header', {
+                    elemClassName: headerClassName,
+                })}
+            >
+                {header}
+            </div>
+            {children}
+        </div>
+    );
+};
```

Comment:
As far as I know, we don't need to wrap sticky items in a container with a position relative. A sticky container will be assigned automatically if the child element is sticky, thus we can make `StickyHeader` component more simple:
```
<div {...rest} {...block()}>
   {children}
</div>
```
What do you think?
https://medium.com/@elad/css-position-sticky-how-it-really-works-54cd01dc2d46

Reply:
In your example, you mean that `{children}` now is a content of sticky element, and the list itself can be completely out of the `StickyHeader` component?

Reply:
yes, children is the sticky element.